ui/src/components/LeftDrawer/LeftDrawer.tsx

```diff
@@ -62,15 +67,51 @@ const Drawer = styled(MuiDrawer, {
   }),
 }));
 
+const AppBar = styled(MuiAppBar, {
+  shouldForwardProp: (prop) => prop !== "open",
+})<AppBarProps>(({ theme, open }) => ({
+  transition: theme.transitions.create(["width", "margin"], {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  width: `calc(${drawerWidthClosed} + 1px)`,
+  height: navbarHeight,
+  right: "auto",
+  border: "none",
+  boxShadow: "none",
+  ...(!open && { zIndex: theme.zIndex.drawer + 1 }),
```

Comment:
that's fucking up the transition when you close it, it didn't find a proper solution for now